service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -156,10 +179,11 @@ private String getLocalAuthorityName(String caseLocalAuthority) {
         return localAuthorityNameLookupConfiguration.getLocalAuthorityName(caseLocalAuthority);
     }
 
-    private String careOrderDetails(int numOfChildren, String caseLocalAuthority) {
-        String childOrChildren = (numOfChildren == 1 ? "child is " : "children are ");
-        return "It is ordered that the " + childOrChildren + "placed in the care of " + getLocalAuthorityName(
-            caseLocalAuthority) + ".";
+    private String careOrderDetails(int numOfChildren, String caseLocalAuthority, OrderTypeAndDocument typeAndDoc) {
```

Comment:
Is this not better as template logic? perhaps not, but should not test it in the service tests?